web/packages/teleport/src/Audit/EventList/EventTypeCell.tsx

```diff
@@ -107,6 +107,7 @@ const EventIconMap: Record<EventCode, any> = {
   [eventCodes.ACCESS_REQUEST_REVIEWED]: Icons.Info,
   [eventCodes.ACCESS_REQUEST_DELETED]: Icons.Info,
   [eventCodes.ACCESS_REQUEST_RESOURCE_SEARCH]: Icons.Info,
+  [eventCodes.ACCESS_REQUEST_EXPIRY]: Icons.Info,
```

Comment:
Should this be `ACCESS_REQUEST_EXPIRED`?

Reply:
On the backend it seems to be "access_request.expire" instead of either expired or expiry. Would you suggest matching the backend  or going with "access_request.expired"?

Reply:
Have updated to be ACCESS_REQUEST_EXPIRED and 'access_request.expire' to match the ACCESS_REQUEST_CREATED and 'access_request.create' convecntion